ads/yandex.js

```diff
@@ -49,13 +49,17 @@ function addToQueue(global, data) {
       data: data.data,
       async: true,
       onRender: () => {
-        // Move adfox queue
-        if (data.isAdfox && global.Ya.adfoxCode.onRender) {
-          global.Ya.adfoxCode.onRender();
+        if (data.onRender) {
```

Comment:
I am not clear how your `render()` function, work. So I want to confirm here what is the `data` object here? Just afraid that `data.onRender` and `data-onError` are both strings.

Reply:
`data` has `onRender` and `onError` callbacks.
There are used only for internal purposes in adfox module and are useless for users.
Added typeOfFunction check if user added them to config as strings. Thanks.